pkg/field/fields.go

```diff
@@ -51,8 +51,8 @@ func (s SchemaField) String() (string, error) {
 	return value, nil
 }
 
-// StringArray retuns the default value as a string array.
-func (s SchemaField) StringArray() ([]string, error) {
+// StringSlice retuns the default value as a string array.
+func (s SchemaField) StringSlice() ([]string, error) {
```

Comment:
Do existing connectors use StringArray or StringArrayField? I know the new name is more accurate, but I'd like to avoid breaking connectors when upgrading their baton-sdk dependency.

Reply:
This field actually doesn't work at all btw